app-antivirus/clamav/clamav-0.105.1.ebuild

```diff
@@ -0,0 +1,236 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LLVM_MAX_SLOT=13
+PYTHON_COMPAT=( python3_{8..11} )
+inherit cmake flag-o-matic llvm python-any-r1 systemd tmpfiles
+
+DESCRIPTION="Clam Anti-Virus Scanner"
+HOMEPAGE="https://www.clamav.net/"
+SRC_URI="https://www.clamav.net/downloads/production/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc clamonacc +clamapp jit libclamav-only milter rar selinux systemd test"
+
+REQUIRED_USE="libclamav-only? ( !clamonacc !clamapp !milter )
+	clamonacc? ( clamapp )
+	milter? ( clamapp )
+	test? ( !libclamav-only )"
+
+RESTRICT="!test? ( test )"
+
+# Require acct-{user,group}/clamav at build time so that we can set
+# the permissions on /var/lib/clamav in src_install rather than in
+# pkg_postinst; calling "chown" on the live filesystem scares me.
+CDEPEND="
+	acct-group/clamav
+	acct-user/clamav
+	app-arch/bzip2
+	dev-libs/json-c:=
+	dev-libs/libltdl
+	dev-libs/libmspack
+	dev-libs/libpcre2:=
+	dev-libs/libxml2
+	dev-libs/openssl:=
+	dev-libs/tomsfastmath:=
+	>=sys-libs/zlib-1.2.2:=
+	virtual/libiconv
+	!libclamav-only? ( net-misc/curl )
+	clamapp? ( sys-libs/ncurses:= net-misc/curl )
+	elibc_musl? ( sys-libs/fts-standalone )
+	jit? ( <sys-devel/llvm-$((${LLVM_MAX_SLOT} + 1)):= )
+	milter? ( mail-filter/libmilter:= )
+	rar? ( app-arch/unrar )
+	test? ( dev-python/pytest )
+"
+
+BDEPEND="
+	virtual/pkgconfig
+	>=virtual/rust-1.56
+	doc? ( app-doc/doxygen )
+	test? (
+		${PYTHON_DEPS}
+		$(python_gen_any_dep 'dev-python/pytest[${PYTHON_USEDEP}]')
+	)
+"
+
+DEPEND="${CDEPEND}
+	test? ( dev-libs/check )"
+
+RDEPEND="${CDEPEND}
+	selinux? ( sec-policy/selinux-clamav )"
+
+python_check_deps() {
+	use test && python_has_version -b "dev-python/pytest[${PYTHON_USEDEP}]"
```

Comment:
You can drop the use test && here now.

Reply:
> It is - `find_package(Valgrind)` is still in the codebase. Besides, we don't generally find Valgrind tests valuable to run in ebuilds.

Sorry, should have been more clear; It's passing tests in the sandbox and although the comment indicates: 

```bash
		# Used to enable some more tests but doesn't behave well in
		# sandbox necessarily(?) + needs certain debug symbols present
		# in e.g. glibc.
```

It doesn't actually seem to enable any additional tests.

I'm happy to put it back in behind the `use test` conditional, I just can't see that it's actually having any measurable impact; I can't think of a better way to validate that than seeing if it throws errors as people test and keyword on systems that aren't mine.